build.sbt

```diff
@@ -0,0 +1,56 @@
+import sbt.Keys._
+
+import org.scalajs.jsenv.selenium.SeleniumJSEnv
+
+val commonSettings: Seq[Setting[_]] = Seq(
+  version := "0.0.1-SNAPSHOT",
+  organization := "org.scala-js",
+  scalaVersion := "2.11.7",
+  scalacOptions ++= Seq("-deprecation", "-feature", "-Xfatal-warnings"),
+
+  homepage := Some(url("http://scala-js.org/")),
+  licenses += ("BSD New",
+      url("https://github.com/scala-js/scala-js-env-selenium/blob/master/LICENSE")),
+  scmInfo := Some(ScmInfo(
+      url("https://github.com/scala-js/scala-js-env-selenium"),
+      "scm:git:git@github.com:scala-js/scala-js-env-selenium.git",
+      Some("scm:git:git@github.com:scala-js/scala-js-env-selenium.git")))
+)
+
+val fatalWarningsSettings = Seq(
+  // The pattern matcher used to exceed its analysis budget before 2.11.5
```

Comment:
I don't think this is a problem for this codebase. Try enabling `-Xfatal-warnings` in all versions.